stdlib/3/builtins.pyi

```diff
@@ -608,7 +608,7 @@ class range(Sequence[int]):
     @overload
     def __init__(self, start: int, stop: int, step: int = 1) -> None: ...
     def count(self, value: int) -> int: ...
-    def index(self, value: int, start: int = 0, stop: int = None) -> int: ...
+    def index(self, value: int) -> int: ...
```

Comment:
This is the one change that isn't only making something non-Optional.  In trying to test the third argument of `range.index`, I found that the function only takes one argument (in spite of `help(range)` suggesting otherwise).